packages/experimental/Avatar/src/Avatar.styling.ts

```diff
@@ -0,0 +1,159 @@
+import { avatarName, AvatarTokens, AvatarSlotProps, AvatarProps } from './Avatar.types';
+import { Theme, UseStylingOptions, buildProps } from '@fluentui-react-native/framework';
+import { defaultAvatarTokens } from './AvatarTokens';
+import { ViewStyle } from 'react-native';
+import { calculateEffectiveSizes, convertCoinColorFluent, getRingThickness } from './Avatar.helpers';
+
+const nameMap: { [key: string]: string } = {
+  start: 'flex-start',
+  center: 'center',
+  end: 'flex-end',
+};
+
+export const stylingSettings: UseStylingOptions<AvatarProps, AvatarSlotProps, AvatarTokens> = {
+  tokens: [defaultAvatarTokens, avatarName],
+  slotProps: {
+    root: buildProps(
+      (tokens: AvatarTokens) => {
+        const { physicalSize } = calculateEffectiveSizes(tokens);
+        const { horizontalIconAlignment, verticalIconAlignment } = tokens;
+        return {
+          style: {
+            flexDirection: 'row',
+            width: physicalSize,
+            height: physicalSize,
+            justifyContent: nameMap[horizontalIconAlignment || 'end'] as ViewStyle['justifyContent'],
+            alignItems: nameMap[verticalIconAlignment || 'end'] as ViewStyle['alignItems'],
+            horizontalIconAlignment,
+            verticalIconAlignment,
+          },
+        };
+      },
+      ['horizontalIconAlignment', 'verticalIconAlignment'],
+    ),
+    initials: buildProps(
+      (tokens: AvatarTokens) => {
+        const { initialsSize } = calculateEffectiveSizes(tokens);
+        return {
+          style: {
+            fontSize: initialsSize,
+          },
+        };
+      },
+      ['initialsSize'],
+    ),
+    initialsBackground: buildProps(
+      (tokens: AvatarTokens) => {
+        const { physicalSize } = calculateEffectiveSizes(tokens);
+        const { backgroundColor, coinColorFluent } = tokens;
+        let effectiveBackgroundColor = backgroundColor;
+        if (coinColorFluent) {
+          effectiveBackgroundColor = convertCoinColorFluent(coinColorFluent);
+        }
+        return {
+          style: {
+            borderRadius: physicalSize / 2,
```

Comment:
I have a minor concern that borderRadius: physicalSize / 2 could produce unexpected results for less-round scale factors e.g. 125% or 175% on win32; could you check the behavior here and see if it creates a proper circle at the available scale factors up to 225%?  I'm less concerned about the other platforms with better float DIP support

I believe `50%` is available as a valid borderRadius value the same as in CSS (though I want to say it isn't implemented on win32...) as a micro means of avoiding layout calculation in JS and leaving correctness to the native platform -- which is to say it may need to be tracked as a git issue if either don't work on win32 🙃

Reply:
I wouldn't block on this issue [and my review isn't blocking at all]

Reply:
Today I was working on adding shape property, so issue with borderRadius is fixed :) I used global tokens for circilar border